docs/develop/parent-to-dialog.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Passing data and messages to a dialog box from its host page
+description: 'Learn how to pass data to a dialog from the host page'
+ms.date: 03/11/2020
+localization_priority: Normal
+---
+
+# Passing data and messages to a dialog box from its host page
+
+Microsoft is now making available for preview new APIs for messaging from the host page (defined [here](dialog-api-in-office-add-ins.md#open-a-dialog-box-from-a-host-page)) to the dialog box.
+
+> [!Important]
+>
+> - The APIs are in preview. They are available to developers for experimentation; but they should not be used in a production add-in. Until this API is released, use the techniques described in [Pass information to the dialog box](dialog-api-in-office-add-ins.md#pass-information-to-the-dialog-box) in production add-ins.
+> - The APIs described in this article require Office 365 (the subscription version of Office). You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features is turned off for that build.
+> - The APIs are only supported in Excel during the preview.
```

Comment:
Won't the APIs only be supported in Excel after the preview too? Or is part of the GA criteria multi-host support?

Reply:
It is uncertain at this time when these APIs will be made available in other hosts. It is possible that will happen during preview, at GA, or even after preview. All we know now is that in the initial preview they are supported only in Excel.